tutorials/distributed-keras-on-cmle/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Distributed Keras on Google Cloud Machine Learning Engine
+description: Learn how to use distributed Keras on Cloud Machine Learning Engine.
+author: corrieelston
+tags: Data Science, Cloud Machine Learning Engine, Keras, Python
+date_published: 2017-09-28
+---
+
+# Distributed Keras on Google Cloud Machine Learning Engine (CMLE)
+
+This solution connects the dots between a number of technologies with the end result of a production-grade model.
```

Comment:
This intro sentence is super-vague. Why not name the technologies? (I know you list them just below, but putting them in the intro paragraph can help people find the paper in the first place.) Also, what does "production-grade model" mean? I'm pretty sure I know what you mean, but it's a pretty subjective idea. What does one look like? How do you know it from a model that isn't "production-grade"?

Reply:
Maybe something like:
This solutions shows you how to use TensorFlow, Keras, and Google Cloud Machine Learning Engine to create a machine learning model that [whatever the distinguishing characteristics of "production-grade" are].
?